README.md

```diff
@@ -52,21 +59,24 @@ The following table lists the configurable parameters of the SFTP server chart a
 | `service.enabled`                  | If true, expose as Service                 | `true`                                      |
 | `service.type`                     | Type of exposed Service                    | `ClusterIP`                                 |
 | `service.port`                     | Port to expose Service                     | `22`                                        |
-| `sftpConfig.username`              | SFTP username                              | `sftp`                                      |
-| `sftpConfig.password`              | SFTP password for user                     | `""`                                        |
-| `sftpConfig.encrypted`             | If true, password is given as hash         | `false`                                     |
-| `sftpConfig.uid`                   | UID of SFTP user                           | `1000`                                      |
-| `sftpConfig.gid`                   | GID of SFTP user                           | `100`                                       |
+| `sftpConfig.users.username`        | SFTP username                              | `sftp`                                      |
```

Comment:
README should be updated to show that the fields under `users` are items in a list of `users`, not fields directly attached to `users`.